extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -145,4 +145,17 @@ private LDAPGuacamoleProperties() {}
 
     };
 
+    /**
+     * The maxResult a ldap search can return
```

Comment:
And "The maximum number of results" would be better still. Just "max results" seems rather wonky to me.